### PR TITLE
Whole Exome Sequencing

### PR DESCRIPTION
**Pull request by SophieS9**
Adding in all changes needed for the whole exome sequencing additions to the pipeline. 

See CC-21-005-GEN for full details on the validation. 

Also tested TSO pipeline to ensure this still functioned correctly using run 210309_NB551319_0184_AHGKGNBGXH.

Linked to update to dragenge_post_processing pipeline - https://github.com/AWGL/dragenge_post_processing/pull/11

### CODE REVIEW

File: DragenGE.sh [hunk 1]
```diff
@@ -16,7 +16,6 @@ version=2.0.0
 ##############################################
 
 pipeline_dir="/data/diagnostics/pipelines/"
-dragen_ref="/staging/resources/human/reference/GRCh37"
```
Comment (josephhalstead) [inline DragenGE.sh:19]: moved to variables files

File: config/IlluminaTruSightOne38/IlluminaTruSightOne.variables [hunk 2]
```diff
@@ -0,0 +1,6 @@
+post_processing_pipeline=dragenge_post_processing
```
Comment (josephhalstead) [inline config/IlluminaTruSightOne38/IlluminaTruSightOne.variables:1]: changed file name to [IlluminaTruSightOne38.variables

**Review by josephhalstead — CHANGES_REQUESTED** [review]
All good - but can we change the names of the files in [config/IlluminaTruSightOne38 to be IlluminaTruSightOne38.variables etc?

**Comment by josephhalstead** [comment]
Also delete sample_cross_contamination_resource_b37.vcf from NonocusWES38 folder

**Comment by SophieS9** [comment]
Changes made - files renamed in IlluminaTruSightOne38 config and removed sample_contamination_b37.vcf from NonocusWES38

**Review by josephhalstead — APPROVED** [review]

**Pull request merged**